app/lib/reducers/ui.js

```diff
@@ -24,7 +24,15 @@ const initial = Immutable({
   fontFamily: 'Menlo, "DejaVu Sans Mono", "Lucida Console", monospace',
   fontSizeOverride: null,
   css: '',
-  termCSS: '',
+  termCSS: '' +
+    'x-row {' +
+    '  text-shadow: 0 0 0 #fff;' +
```

Comment:
You mean the color set for the text or the color set for the cursor? If the former, you have to set it to `foregroundColor`, and make sure that when `foregroundColor` changes the css gets reloaded in `componentWillReceiveProps`